### PR TITLE
flake8-sfs v0.0.2 receipe

### PR DESCRIPTION
**Pull request by peterjc**
Based on receipt for another pure-Python flake8 plugin, flake8-black, this adds a recipe for my newest plugin, flake8-sfs

https://github.com/peterjc/flake8-sfs
https://pypi.org/project/flake8-sfs

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by peterjc** [comment]
@conda-forge/help-python ready for review?

File: recipes/flake8-sfs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "flake8-sfs" %}
+{% set version = "0.0.2" %}
+{% set sha256 = "aeada40b526bf49d443f3f21483705dd8fa958c23809671e2ea8ab8cc1eeb4ab" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (npavlovikj) [inline recipes/flake8-sfs/meta.yaml]: `fn` is not required/used anymore with the newer `conda-builda` versions, so please remove it.

**Comment by npavlovikj** [comment]
@peterjc , I left a minor comment - we can merge the recipe afterwards.

File: recipes/flake8-sfs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
```
Comment (npavlovikj) [inline recipes/flake8-sfs/meta.yaml]: This should be `host` instead (`build` is used when there are compilers used).

File: recipes/flake8-sfs/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    - python >=3.6
+    - pip
+  run:
+    - python
```
Comment (npavlovikj) [inline recipes/flake8-sfs/meta.yaml]: Use `python >=3.6` instead.

**Comment by npavlovikj** [comment]
I am sorry @peterjc , I missed 2 minor things earlier.

File: recipes/flake8-sfs/meta.yaml (hunk 3 above)
Comment (peterjc) [reply recipes/flake8-sfs/meta.yaml]: Good catch, that was inconsistent and important.

Comment (npavlovikj) [reply recipes/flake8-sfs/meta.yaml]: I think you changed `flake8` version instead of `python` by mistake :)

Comment (peterjc) [reply recipes/flake8-sfs/meta.yaml]: Thank you, yes.

While I'm at I, might as well make the requirement for at least flake8 v3 explicit too.

**Comment by npavlovikj** [comment]
All green and good @peterjc - thank you for contributing 🎉

**Comment by peterjc** [comment]
Thank you! Very useful feedback, it will help update another receipt too:
https://github.com/conda-forge/flake8-black-feedstock/pull/2

**Pull request merged**